community/sway/etc/skel/.config/nwg-wrapper/style.css

```diff
@@ -1,17 +1,18 @@
 window {
-        background-color: rgba (0, 0, 0, 0.0);
+	background-color: rgba (0, 0, 0, 0.0);
+	font-size: 20px;
```

Comment:
doesn't this work with relative values like `medium` or `large`? I think this would be better for cross-device compatibility (if it works).

Reply:
https://github.com/nwg-piotr/nwg-wrapper/issues/9

Reply:
`large` is still quite small on my monitor, but at least a little better. `xx-large` actually works well, can you try on your machine and then we make an educated guess for a sensible default?

Reply:
I think this could get very tricky / we probably then are better off with a fixed value. large would already be huge on regular horizontal 4k.

Reply:
After setting sway output `scale` to 2, `large` is ideal.